app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
```

Comment:
Let's format all of the import statements in the same way to improve code readability

```python
from app.errors import (
    NotWearingMaskError,
    NotVaccinatedError,
     OutdatedVaccineError,
)
```